app/views/applications/_form.html.haml

```diff
@@ -105,6 +105,7 @@
               .container.schedule_items_row.w-100.my-4.p-3.border.border-secondary
                 .row
                   .col-md-4
+                    // TODO - whatever is todo??? 
```

Comment:
If I recall correctly a `Todo` is an item that represents things to be done on some schedule. This model should be called `ScheduleItem` I guess.

Reply:
I'll rename it then. "todo" is very non-descriptive

Reply:
Okay, resolve this comment once done. Remember you have to rename:
1. Model.
2. Table in the database.
3. All occurennces of `:todo` in relations in other models and tables.

Reply:
@TeachMeHow Did you finish that one?